### PR TITLE
docs: correct getting started version

### PR DESCRIPTION
**Pull request by kassianah**
https://app.shortcut.com/mineiros/story/5060/correct-getting-started-example-version

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,8 +55,8 @@ full access to AWS Simple Storage Service (S3)
 
 ```hcl
 module "role-s3-full-access" {
-  source  = "mineiros-io/iam-role/aws"
-  version = "~> 0.6.0"
+  source = "mineiros-io/iam-role/aws"
+  version = "~> 0.6.1"
```
Comment (mariux) [inline README.md]: please align indent of `=` right

File: README.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ See [variables.tf] and [examples/] for details and use-cases.
 
   Default is `{}`.
 
-- [**`module_depends_on`**](#var-module_depends_on): *(Optional `list(any)`)*<a name="var-module_depends_on"></a>
+- [**`module_depends_on`**](#var-module_depends_on): *(Optional `list(object)`)*<a name="var-module_depends_on"></a>
```
Comment (mariux) [inline README.md]: how did that happen? please update from template ;)

**Review by mariux — DISMISSED** [review]

File: README.md (hunk 2 above)
Comment (kassianah) [reply README.md]: Terradoc handling things differently,, Have updated it to be correct.

File: README.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ full access to AWS Simple Storage Service (S3)
 ```hcl
 module "role-s3-full-access" {
   source  = "mineiros-io/iam-role/aws"
-  version = "~> 0.6.0"
+  version = "~> 0.6.1"
```
Comment (soerenmartius) [inline README.md:59]: we don't need to specify a patch when using a range according to semver https://semver.org/

Comment (soerenmartius) [inline README.md:59]:
```suggestion
  version = "~> 0.6.0"
```

**Review by soerenmartius — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (kassianah) [reply README.md:59]: Good point, you've said this before and I forgot.

Comment (kassianah) [reply README.md:59]: In that case I will close this PR since it's no-longer necessary.

**Pull request closed without merging**